client/js/libs/handlebars/index.js

```diff
@@ -0,0 +1,10 @@
+import "./colorClass";
+import "./date";
+import "./diff";
+import "./equal";
+import "./modes";
+import "./parse";
+import "./roundBadgeNumber";
+import "./tojson";
+import "./tz";
+import "./users";
```

Comment:
I sure wish there was a way to do `import "./*";` but apparently there is no such thing.